### DISCUSSION TITLE
umi升级到3.5 报错：pathname.match is not a function

### DISCUSSION THREAD

**Discussion by lkzwc** [umijs/umi#10103] [Q&A]
![image](https://user-images.githubusercontent.com/84896877/208345245-1e5ee823-0b12-44de-8ca7-127a9f340905.png)
![image](https://user-images.githubusercontent.com/84896877/208345631-ecdfc20c-bc89-4f26-bfb3-1e2c396a9e63.png)

**Comment by mmmaaaa** [comment]
请问你这个问题后面解决了吗

**Comment by ly4236** [comment]
清空了npm缓存，删除了node_modules，然后重新安装就好了。
